Simplenote/Controllers/WidgetController.swift

```diff
@@ -0,0 +1,37 @@
+import WidgetKit
+
+struct WidgetController {
+    @available(iOS 14.0, *)
+    static func resetWidgetTimelines() {
+        WidgetCenter.shared.getCurrentConfigurations { result in
+            guard case .success(let widgets) = result else {
+                return
+            }
+
+            if widgets.contains(where: { (widget) -> Bool in
+                if widget.configuration as? NoteWidgetIntent != nil {
+                    return true
+                }
+
+                // Add check for ListWidgetIntent when created later
+
+                return false
+            }) {
+                NSLog("Reloading all widgets")
+                WidgetCenter.shared.reloadAllTimelines()
+            }
+        }
+    }
+
+    static func syncWidgetDefaults(isLoggedIn loggedIn: Bool) {
```

Comment:
```suggestion
    static func syncWidgetDefaults(authenticated: Bool) {
```

Really, really, like, really nipticky!!